be/src/exprs/vectorized/dictmapping_expr.h

```diff
@@ -45,6 +55,8 @@ class DictMappingExpr final : public Expr {
     }
 
 private:
+    std::shared_ptr<std::once_flag> _rewrite_once_flag = std::make_shared<std::once_flag>();
```

Comment:
can once_flag be an ordinary flag?

Reply:
once_flag don't have a copy constructor. but we should implement clone method for Expr
```
Expr* clone(ObjectPool* pool) const override { return pool->add(new DictMappingExpr(*this)); }
```